### PR TITLE
Feature: System Color Scheme

### PR DESCRIPTION
**Pull request by tunds**
Added system settings for light, dark, and system theme for the entire app, also did some refactoring on the setting screen to break out views shown in the list.
Simulator Screen Recording - iPhone 11 - 2022-05-05 at 15 49 18

### CODE REVIEW

**Comment by chFlorian** [comment]
Good stuff!

**Review by adamrushy — COMMENTED** [review]
Few minor comments :]

File: Social Contributor.xcodeproj/xcuserdata/tunde.adegoroye.xcuserdatad/xcschemes/xcschememanagement.plist [hunk 1]
```diff
@@ -0,0 +1,14 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
+<plist version="1.0">
+<dict>
+	<key>SchemeUserState</key>
+	<dict>
+		<key>Social Contributor.xcscheme_^#shared#^_</key>
+		<dict>
```
[the remark this answers is not in the document]
Comment (chFlorian) [reply Social Contributor.xcodeproj/xcuserdata/tunde.adegoroye.xcuserdatad/xcschemes/xcschememanagement.plist:8]: Maybe add these files to .gitignore?

**Comment by tunds** [comment]
@adamrushy All those issues should be resolved now 👍🏾

**Review by adamrushy — APPROVED** [review]
LGTM ✅

**Pull request merged**